app/src/main/java/com/teamtwo/apilol/model/LOLService.kt

```diff
@@ -5,10 +5,10 @@ interface LOLService {
     /* Architect Codders Example
 
     @GET("discover/movie?sort_by=popularity.desc")
-    fun listPopularMoviesAsync(
+    suspend fun listPopularMoviesAsync(
         @Query("api_key") apiKey: String,
         @Query("region") region: String
-    ): Deferred<MovieDbResult>
+    ): MovieDbResult
```

Comment:
Si ponemos el objeto de respuesta del servicio con el wrapper `Response<T>`, además del objeto obtenemos toda la información de la respuesta, como códigos HTTP para poder saber si la respuesta ha sido correcta.
```suggestion
    ): Response<MovieDbResult>
```